tests/tests_certificate.yml

```diff
@@ -23,7 +23,12 @@
   vars:
     cockpit_packages: minimal
   roles:
-    - linux-system-roles.cockpit
+    - role: linux-system-roles.cockpit
+      public: true
```

Comment:
This `public` is an unrelated change, and should be a separate commit. Same for the other instances below.

Reply:
I haven't checked, but it looks like check_firewall_selinux.yml requires role private variables in order to check the values? If so, then I think `public: true` is justified

Reply:
Right. `check_firewall_selinux.yml` needs it. And I'm going to drop the unnecessary `public: true` other than `tests_config.yml`, `tests_packages_full.yml`, and `tests_port.yml` along with `check_firewall_selinux.yml`.

Reply:
@nhosoi : I see -- I didn't realize that it's really the new tasks/* files which require `public:`. That's fine then.